lxd/container.go

```diff
@@ -62,6 +63,23 @@ func init() {
 
 		return device.Instance(container), nil
 	}
+
+	// Expose instanceLoadById to the backup package converting the response to an Instance.
+	// This is because container types are defined in the main package and are not importable.
+	backup.InstanceLoadByID = func(s *state.State, id int) (backup.Instance, error) {
```

Comment:
@monstermunchkin cool, can you just link to instanceLoadByID in main rather than duplicating it's implementation though?

Reply:
Fixed